src/db_queries/auth.ts

```diff
@@ -43,16 +43,17 @@ export class Auth {
       response.status(400).send({'message': 'Please enter a valid email address'});
       return ;
     }
-    
+    const getUsersQuery = 'SELECT * FROM users WHERE email = $1';
+    const client: pg.PoolClient = await pool.connect();
     try {
-      const getUsersQuery = 'SELECT * FROM users WHERE email = $1';
-      let { rows }: pg.QueryResult = await pool.query(getUsersQuery, [email]);
+      await client.query("BEGIN");
```

Comment:
I'm not sure if transactions are necessary for single statements (https://stackoverflow.com/questions/1171749/what-does-a-transaction-around-a-single-statement-do) or for SELECT queries (in the 13.2.1 section of this page: https://www.postgresql.org/docs/9.1/transaction-iso.html it says: "However, SELECT does see the effects of previous updates executed within its own transaction, even though they are not yet committed. Also note that two successive SELECT commands can see different data, even though they are within a single transaction, if other transactions commit changes during execution of the first SELECT."). 
The above applies to other SELECT queries and single statements in this PR.

Reply:
I have made this change as discussed.